service/src/main/resources/api/openapi.yml

```diff
@@ -382,6 +382,31 @@ paths:
         "500":
           $ref: "#/components/responses/ServerError"
 
+  /api/workspaces/{workspaceId}/cromwell/parseInputsAndZip/{gcsPath}:
+    parameters:
+      - $ref: "#/components/parameters/WorkspaceId"
+      - $ref: "#/components/parameters/GcsPath"
```

Comment:
nit: I feel like this probably doesn't need a ref, could just define the field right here. But its prob fine